api/src/main/java/org/codice/ddf/admin/api/handler/report/ProbeReport.java

```diff
@@ -74,8 +74,18 @@ public ProbeReport probeResults(Map<String, Object> results) {
         return probeResults;
     }
 
+    public <T> T getProbeResult(String key) {
+        if(probeResults.get(key) != null) {
```

Comment:
This makes sense. The other place we were using this process involved primitives. I'll make the change and confirm everything works.